packages/terra-i18n/tests/nightwatch/i18n-spec.js

```diff
@@ -12,6 +12,8 @@ const defaultTranslations = enTransaltions;
 
 const screenshot = require('terra-toolkit').screenshot;
 
+const pauseInms = 1000;
```

Comment:
Assert failure before required locale data, it can randomly happen, like in [here](https://travis-ci.org/cerner/terra-core/builds/213563117#L2394)